lib/screens/home/home.dart

```diff
@@ -53,17 +54,17 @@ class _HomeScreenState extends State<HomeScreen> {
 
   checkLoginStatus() async {
     sharedPreferences = await SharedPreferences.getInstance();
-    if (sharedPreferences.getString("token") == null) {
-      Navigator.of(context).pushAndRemoveUntil(
-          MaterialPageRoute(builder: (BuildContext context) => Login()),
-          (Route<dynamic> route) => false);
-    } else {
-      setState(() {
-        name = sharedPreferences.getString('name');
-        mobile = sharedPreferences.getString('mobile');
-        email = sharedPreferences.getString('email');
-      });
-    }
+    // if (sharedPreferences.getString("token") == null) {
+    //   Navigator.of(context).pushAndRemoveUntil(
+    //       MaterialPageRoute(builder: (BuildContext context) => Login()),
+    //       (Route<dynamic> route) => false);
+    // } else {
+    setState(() {
+      name = 'Mohamed'; // sharedPreferences.getString('name');
+      mobile = '01002266555'; // sharedPreferences.getString('mobile');
+      email = 'mohamed@gmail.com'; //sharedPreferences.getString('email');
+    });
+    // }
```

Comment:
Why?
the state should be dynamic based on the logged-in user, You're making it static!